Specs/Core/EllipsoidRhumbLineSpec.js

```diff
@@ -0,0 +1,581 @@
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON3);
+
+        rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, ninetyDegrees, distance, ellipsoid);
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON3);
+
+        rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, ninetyPointOneDegrees, distance, ellipsoid);
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON3);
+
+        rhumb1 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, ninetyPointZeroTwoDegrees, distance, ellipsoid);
+        rhumb2 = new EllipsoidRhumbLine(initial, rhumb1.end, ellipsoid);
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON3);
+    });
+
+    it('test sphereoid across meridian', function() {
+        var ellipsoid = Ellipsoid.WGS84;
+        var fifteenDegrees = Math.PI / 12;
+        var initial = new Cartographic(-fifteenDegrees, 0.0);
+        var final = new Cartographic(fifteenDegrees, 0.0);
+        var distance = ellipsoid.maximumRadius * 2 * fifteenDegrees;
+
+        var rhumb1 = new EllipsoidRhumbLine(initial, final, ellipsoid);
+        var rhumb2 = EllipsoidRhumbLine.fromStartHeadingDistance(initial, CesiumMath.PI_OVER_TWO, distance, ellipsoid);
+
+        expect(rhumb1.heading).toEqualEpsilon(rhumb2.heading, CesiumMath.EPSILON12);
+        expect(rhumb1.surfaceDistance).toEqualEpsilon(rhumb2.surfaceDistance, CesiumMath.EPSILON6);
+    });
+
+    it('test across IDL with -PI to PI range of longitude', function() {
+        var ellipsoid = Ellipsoid.WGS84;
+        var fifteenDegrees = Math.PI / 12.0;
+        var initial = new Cartographic(-CesiumMath.PI + fifteenDegrees, 0.0);
+        var final = new Cartographic(CesiumMath.PI - fifteenDegrees, 0.0);
+
+        var distance = ellipsoid.maximumRadius * 2 * fifteenDegrees;
+
+        var rhumb1 = new EllipsoidRhumbLine(initial, final, ellipsoid);
+        var rhumb2 = new EllipsoidRhumbLine.fromStartHeadingDistance(initial, 3.0 * CesiumMath.PI_OVER_TWO, distance, ellipsoid);
+
+        expect(-CesiumMath.PI_OVER_TWO).toEqualEpsilon(rhumb1.heading, CesiumMath.EPSILON12);
+        expect(CesiumMath.PI / 6 * ellipsoid.maximumRadius).toEqualEpsilon(rhumb1.surfaceDistance, CesiumMath.EPSILON6);
```

Comment:
CesiumMath.PI / 6 * ellipsoid.maximumRadius -> distance? Or at least use the same math code so it's a little more clear.